Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -221,11 +250,46 @@ class SyncNowSetting: WithAccountSetting {
     override func onClick(_ navigationController: UINavigationController?) {
         NotificationCenter.default.post(name: Notification.Name(rawValue: SyncNowSetting.NotificationUserInitiatedSyncManually), object: nil)
         profile.syncManager.syncEverything(why: .syncNow)
+        
+        if AppConstants.MOZ_FXA_PREFERENCE {
+            profile.getAccount()?.updateProfile()
```

Comment:
Why do this? As you've written above we already fetch the profile each time we advance the account object… so this'll do it twice.